tex/generic/pgf/systemlayer/pgfsys-dvisvgm.def

```diff
@@ -127,7 +127,7 @@
   \dp#1=0pt%
   \leavevmode%
   \pgf@xa=\pgf@trimleft@final\relax  \ifdim\pgf@xa=0pt \else\kern\pgf@xa\fi%
-  \raise-\pgf@ya\hbox{\ifpgf@sys@svg@inpicture\else\special{dvisvgm:bbox \pgf@sys@tonumber\pgf@picmaxx\space\pgf@sys@tonumber\pgf@picmaxy}\fi\box#1}%
+  \raise-\pgf@ya\hbox{\ifpgf@sys@svg@inpicture\box#1\else\special{dvisvgm:bbox \pgf@sys@tonumber\pgf@picmaxx\space\pgf@sys@tonumber\pgf@picmaxy}\special{dvisvgm:bbox lock}\box#1\special{dvisvgm:bbox unlock}\fi}%
```

Comment:
For shorter code line hence better readability, maybe
```suggestion
  \raise-\pgf@ya\hbox{%
    \ifpgf@sys@svg@inpicture
      \box#1%
    \else
      \special{dvisvgm:bbox \pgf@sys@tonumber\pgf@picmaxx\space\pgf@sys@tonumber\pgf@picmaxy}%
      \special{dvisvgm:bbox lock}%
      \box#1%
      \special{dvisvgm:bbox unlock}%
    \fi}%
```

Reply:
Thanks for the suggested edit. May I add `%` to protect line endings of lines containing `\ifpgf@...` and `\else`?